### DISCUSSION TITLE
Unique Item apply wrong between different sever

### DISCUSSION THREAD

**Discussion by masking12** [lonqie/SchaleDB#96] [Ideas and Suggestions]
Hibiki‘s unique item is just installed only in Japanese server, but Detailed Character Stats in CN or Global will  incorrectly use this item.
This situation also appears at other students.
![screenshot](https://github.com/user-attachments/assets/26075d72-a1f5-4701-8c8c-cf2e18cc49f4)

**Comment by lonqie** [comment]
This is being tracked in #97
Please open issues and not discussions for website bugs
